translations/atrium-translations-en_GB/src/commonMain/kotlin/ch/tutteli/atrium/translations/DescriptionListLikeExpectation.kt

```diff
@@ -8,5 +8,8 @@ import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
  */
 enum class DescriptionListLikeExpectation(override val value: String) : StringBasedTranslatable {
     /** @since 0.18.0 */
-    INDEX_OUT_OF_BOUNDS("❗❗ index out of bounds")
+    INDEX_OUT_OF_BOUNDS("❗❗ index out of bounds"),
+
+    /** @since 1.2.0 */
+    EMPTY("empty"),
```

Comment:
I suggest we introduce a failure text which makes it clearer that the last element was not actually "empty" but the list was empty:
```suggestion
    WAS_EMPTY("❗❗List was empty"),
```
(for german: WAS_EMPTY("❗❗List war leer")

Other suggestions are welcome

Reply:
I'm not sure about "was". What do you think about "IS_EMPTY"?